### PR TITLE
try codecov

### PR DESCRIPTION
**Pull request by Carreau**
I just tried, and it seem to work [for me](https://codecov.io/github/Carreau/nbgrader?ref=master). If you get annoyed by coverall, it seems easy to switch.

### CODE REVIEW

**Comment by jhamrick** [comment]
Sure, seems like maybe it will be less finicky than coveralls.

File: .travis.yml [hunk 1]
```diff
@@ -16,6 +16,7 @@ addons:
     - pandoc
 before_install:
   - pip install invoke
+  - pip install codecov
```
Comment (jhamrick) [inline .travis.yml]: Could you instead put codecov in the dev-requires in flit.ini and remove coveralls?

File: tasks.py [hunk 2]
```diff
@@ -152,6 +152,7 @@ def tests(group='all', skip=None):
 def after_success(group):
     if group in ('python', 'js'):
         run('coveralls')
+        run("codecov")
```
Comment (jhamrick) [inline tasks.py]: Could you remove the coveralls line?

**Comment by jhamrick** [comment]
Added some comments, or I can also just push to your branch since I have access to your repo (but it's your master branch, so not sure if you want that).

**Comment by Carreau** [comment]
@emilienschultz want to play with github, I gave him access to my repo, if he can fix that before me...

**Comment by Carreau** [comment]
Done, also changed badge in Readme.

**Comment by jhamrick** [comment]
Thanks!

**Pull request merged**